code/dynamic_programming/coin_change/coinchange.cpp

```diff
@@ -4,52 +4,33 @@ using namespace std;
 /*
  * Part of Cosmos by OpenGenus Foundation
 */
+#define MAX 100
```

Comment:
Use a `const int` instead of a macro. Macros are generally not recommended in C++.

Reply:
Will do it.